utils/ChromiumDownloader.js

```diff
@@ -82,11 +84,12 @@ module.exports = {
   /**
    * @param {string} platform
    * @param {string} revision
+   * @param {string} [downloadHost]
    * @param {?function(number, number)} progressCallback
    * @return {!Promise}
    */
-  downloadRevision: function(platform, revision, progressCallback) {
-    let url = downloadURLs[platform];
+  downloadRevision: function(platform, revision, downloadHost, progressCallback) {
+    let url = getDownloadUrl(platform, downloadHost);
```

Comment:
We already do formatting of download urls with `util.format`. I'd prefer this to be done in the same way, e.g.:

```js
const downloadURLs = {
  // Notice the %s in the very beginning
  linux: '%s/chromium-browser-snapshots/Linux_x64/%d/chrome-linux.zip',
  // ...
}
```

And later, every `util.format` will pass in HOST:

```js
url = util.format(url, host, revision);
```

Reply:
How did I miss that, yeah it's clear how to do that.